src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -5084,7 +5084,7 @@ internal enum ErrorCode
             ERR_ExpressionTreeContainsBadCoalesce = 845,
             ERR_ArrayInitializerExpected = 846,
             ERR_ArrayInitializerIncorrectLength = 847,
-            ERR_OverloadRefOutCtor = 851,
+            ERR_OverloadRefKindCtor = 851,
```

Comment:
Ah. Missed it there. Will fix now.

---
In reply to: [104493299](https://github.com/dotnet/roslyn/pull/17547#discussion_r104493299) [](ancestors = 104493299)